jena-arq/src/main/java/org/apache/jena/riot/writer/WriterLib.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.riot.writer;
+
+import org.apache.jena.riot.RIOT;
+import org.apache.jena.sparql.util.Context;
+
+/** Package-scoped utilities */
+/*package*/ class WriterLib {
+
+    static DirectiveStyle dftDirectiveStyle = DirectiveStyle.AT;
```

Comment:
If it's not changed elsewhere, maybe it could be `final` too? Otherwise it could lead to inconsistent results if multiple thread access `directiveStyle` I think? (as it returns its value there)

Reply:
Yes. Done.